### DISCUSSION TITLE
VPN blocks sync

### DISCUSSION THREAD

**Discussion by SWBZA** [abraunegg/onedrive#1249] [Q&A] [answered]
Hi there.

Sync works fine, but not while I am on VPN.  My VPN can only whitelist a port or IP address.  Is there one that I can use that will open up / unblock this app from contacting and syncing with Microsoft?

Thanks in advance,
Schalk

**Comment by abraunegg** [comment, marked as answer]
@SWBZA 
> Sync works fine, but not while I am on VPN. My VPN can only whitelist a port or IP address. Is there one that I can use that will open up / unblock this app from contacting and syncing with Microsoft?

You need to investigate [VPN split tunneling](https://www.google.com/search?q=vpn+split+tunneling+linux&rlz=1C1SQJL_enAU887AU887&oq=vpn+split+tunneling+linux&aqs=chrome..69i57j0i22i30.7872j1j7&sourceid=chrome&ie=UTF-8).
